src/discovery-app.js

```diff
@@ -159,12 +166,9 @@ export class DiscoveryApp extends (navigator(router(FetchMixin(FeatureMixin(Rout
 	async _tokenChanged() {
 		this._initializeToken(this.token);
 
-		//Resolve the token
-		this._getToken().then((token) => {
-			this.resolvedToken = token;
-		});
+		this.resolvedToken = await this._getToken();
+		this. _discoverCookieLoaded = await this.fetchCookie(this.resolvedToken);
```

Comment:
There's a space between `this.` and `_discoverCookieLoaded` that you should remove, don't know how lint didn't catch this.
```suggestion
		this._discoverCookieLoaded = await this.fetchCookie(this.resolvedToken);
```